charts/stardog/README.md

```diff
@@ -58,6 +58,20 @@ to run the steps on the Stardog home directories in the PVCs.
 See the [Stardog documentation](https://www.stardog.com/docs/#_upgrading_the_cluster)
 for instructuions on how to upgrade Stardog Cluster.
 
+### Upgrading from 1.x to 2.x
+
+Version 2.x of the charts deploys ZooKeeper 3.5 instead of 3.4. Before upgrading the
+charts make sure you have upgraded to Stardog 7.4.2 or later, which includes preview
+support for ZooKeeper 3.5.
+
+After upgrading Stardog you can upgrade the charts by taking down your full deployment
+of Stardog and ZooKeeper pods (but do not delete the PVCs with Stardog home data) by
+following this process:
+- Stop all traffic and updates to your cluster or wait for them to end
+- Backup Stardog home and copy the backup out of the k8s environment
```

Comment:
Can we provide a little more help here?  I know there are a few ways to do this that might be k8s deployment specific but I think we could give them a touch more to go on.

Reply:
Sounds good. Added more detail and some links to Stardog docs.